packages/csi-wekafs/package.yaml

```diff
@@ -1,2 +1,2 @@
 url: https://weka.github.io/csi-wekafs/csi-wekafsplugin-0.6.4.tgz
-packageVersion: 00
+packageVersion: 01
```

Comment:
This change was likely caused due to the charts-build-scripts bump but was not caught since the version had changed.

The change introduced in this version is simply that the description is now split across two lines:

https://github.com/rancher/partner-charts/blob/b3d1a2c9fc0bb5af08c76d45e4f5e0344f6b02fd/charts/csi-wekafs/csi-wekafsplugin/0.6.400/Chart.yaml#L6

https://github.com/rancher/partner-charts/blob/7738c4a710973985565e4fa8ba171b6276995eb8/charts/csi-wekafs/csi-wekafsplugin/0.6.400%2Bup0.6.4/Chart.yaml#L6-L7